automod/rules/reposts.go

```diff
@@ -0,0 +1,42 @@
+package rules
+
+import (
+	"time"
+
+	"github.com/bluesky-social/indigo/automod"
+	"github.com/bluesky-social/indigo/automod/countstore"
+)
+
+var repostDailyThreshold = 500
+
+var _ automod.RecordRuleFunc = TooManyRepostRule
+
+// looks for accounts which do frequent reposts
+func TooManyRepostRule(c *automod.RecordContext) error {
+
+	did := c.Account.Identity.DID.String()
+	// Don't bother checking reposts from accounts older than 30 days
+	if c.Account.Private != nil {
+		age := time.Since(c.Account.Private.IndexedAt)
+		if age > 30*24*time.Hour {
+			return nil
+		}
+	}
+
+	switch c.RecordOp.Collection {
+	case "app.bsky.feed.post":
+		c.Increment("post", did)
```

Comment:
I thought we already had a counter like this elsewhere, but guess we don't!

Reply:
I thought we'd have one too but couldn't find one.